src/reducers/versions.spec.tsx

```diff
@@ -268,4 +290,111 @@ describe(__filename, () => {
       expect(dispatch).not.toHaveBeenCalled();
     });
   });
+
+  describe('fetchVersionFile', () => {
+    it('calls getVersion', async () => {
+      const version = fakeVersion;
+      const _getVersion = jest.fn().mockReturnValue(Promise.resolve(version));
+
+      const addonId = 123;
+      const versionId = version.id;
+      const path = 'some/path.js';
+
+      const { store, thunk } = thunkTester({
+        createThunk: () =>
+          fetchVersionFile({ _getVersion, addonId, versionId, path }),
```

Comment:
You're calling `fetchVersionFile` multiple times in this test suite. It seems like a candidate for a helper function.